pkg/host/syscalls_linux.go

```diff
@@ -320,6 +320,23 @@ func onlySandboxNoneOrNamespace(sandbox string) (bool, string) {
 	return true, ""
 }
 
+func isSupportedIOUring() (bool, string) {
+	// TODO: Ensure /proc/kallsysms is read before calling this function.
+	if !testFallback && len(kallsymsSyscallSet) != 0 {
```

Comment:
This duplicates the last half of the isSupported function.
It will be better to factor out that part of isSupported into a separate function and then call it here, or probably right in the switch above because it will be just 1 call:
```
isSupportedSyscall(target.SyscallMap["io_uring"])
```

Reply:
[Fixed](https://github.com/google/syzkaller/pull/1926/commits/b9950dc9fb2fd181607037210a85b221ec09d335)